components/input/apiExamples/minLength.html

```diff
@@ -1,9 +1,9 @@
-<auro-input required type="number" minlength="1">
-  <slot slot="label">Account number</slot>
-  <slot slot="helptext">Please enter your 12 digit account number.</slot>
+<auro-input required minlength="12" setCustomValidityTooShort="Please enter a full voucher code.">
```

Comment:
I'd like to suggest a much smaller number. Since this is for demo purposes it would be nice to not have to type 13 characters to see the behavior. 3 or 4 would be sufficient.